### PR TITLE
Fix indowebnovel info and chapters"

### PR DESCRIPTION
**Pull request by yogasw**
i fix issue when get images, titles and chapter lists

### CODE REVIEW

File: sources/id/indowebnovel.py [hunk 1]
```diff
@@ -46,12 +46,12 @@ def read_novel_info(self):
                     "id": chap_id,
                     "volume": vol_id,
                     "url": self.absolute_url(a["href"]),
-                    "title": a.text.strip() or ("Chapter %d" % chap_id),
+                    "title": a.get_text(" - ").strip() or ("Chapter %d" % chap_id),
```
Comment (dipu-bd) [inline sources/id/indowebnovel.py:49]: This can be used: `a.get_text(" - ", strip=True)`

Comment (dipu-bd) [reply sources/id/indowebnovel.py:49]: Please send pull request to `dev` branch

Comment (yogasw) [reply sources/id/indowebnovel.py]: I will create a new PR, because I previously took it from the main branch.

Comment (yogasw) [reply sources/id/indowebnovel.py]: I created a new PR here
https://github.com/dipu-bd/lightnovel-crawler/pull/1949/files

Comment (dipu-bd) [reply sources/id/indowebnovel.py]: thanks

**Pull request closed without merging**